### PR TITLE
Hk fixes

### PR DESCRIPTION
**Pull request by hkalodner**
- Add owner controls to rollup contract
- Enable removing of old stakes

### CODE REVIEW

File: packages/arb-bridge-eth/contracts/rollup/ArbRollup.sol [hunk 1]
```diff
@@ -283,6 +287,46 @@ contract ArbRollup is IArbRollup, Cloneable, NodeGraph, Staking {
         safeSelfDestruct(msg.sender);
     }
 
+    function ownerToggleStakerAllowListed(bool shouldRequire) external onlyOwner {
```
Comment (edfelten) [inline packages/arb-bridge-eth/contracts/rollup/ArbRollup.sol:290]: I think this should be called `ownerToggleStakerAllowList`.

**Review by joshuacolvin0 — APPROVED** [review]
LGTM

**Review by edfelten — DISMISSED** [review]
Looks good.  One small naming suggestion, but I'm not going to hold this up over that, given that this code is probably obsolete anyway.

File: packages/arb-bridge-eth/contracts/rollup/ArbRollup.sol (hunk 1 above)
Comment (hkalodner) [reply packages/arb-bridge-eth/contracts/rollup/ArbRollup.sol:290]: I'd change this, but I already deployed it

**Pull request merged**